karma.conf.js

```diff
@@ -67,8 +67,13 @@ module.exports = function(config) {
 
         // start these browsers
         // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-        browsers: ["PhantomJS"],
-
+        browsers: ["HeadlessChrome"],
+        customLaunchers: {
+          HeadlessChrome: {
+            base: 'ChromeHeadless',
+            flags: [ '--no-sandbox', ],
```

Comment:
Do you mind explaining why this flag is necessary?

Reply:
Leaving this link to the [Travis CI docs](https://docs.travis-ci.com/user/chrome#sandboxing) here for posterity. Thanks for the explanation!